classcreater.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require 'config.php';
+$title = $_POST['title'];
+$classcode = $_POST['classcode'];
+$passcode = $_POST['passcode'];
+$q = $mysqli->query("SELECT * FROM class WHERE code = '$classcode'");
+if($q->num_rows>0)
+{
+  $_SESSION['message'] = 'class with this classcode already registered!';
+  header("location: displaymessage.php");
```

Comment:
Where is displaymessage.php?

Reply:
to be implemented for showing these messages